src/snowflake/cli/api/console/console.py

```diff
@@ -8,10 +8,10 @@
 from snowflake.cli.api.console.abc import AbstractConsole
 from snowflake.cli.api.console.enum import Output
 
-PHASE_STYLE: Style = Style(color="grey93", bold=True)
-STEP_STYLE: Style = Style(color="grey89", italic=True)
-INFO_STYLE: Style = Style(color="black")
-IMPORTANT_STYLE: Style = Style(color="red", bold=True, italic=True)
+PHASE_STYLE: Style = Style(bold=True)
+STEP_STYLE: Style = Style(italic=True)
+INFO_STYLE: Style = Style()
+IMPORTANT_STYLE: Style = Style(bold=True, italic=True)
```

Comment:
Can you show an example?

Reply:
Examples with light and dark background before and after in description

Reply:
<img width="1192" alt="light_bg" src="https://github.com/Snowflake-Labs/snowflake-cli/assets/152898938/2cddaf5d-31f2-4ca0-b5f2-0284c7921709">
Below are examples with basic colours.

<img width="1269" alt="dark_bg" src="https://github.com/Snowflake-Labs/snowflake-cli/assets/152898938/dc862455-1be7-441c-b513-66e71f69515e">

Still this will look different on user terminals. It all depends on profiles, transparency settings. There is [no reliable way](https://stackoverflow.com/questions/69348880/is-it-possible-to-use-background-aware-color-choices/69352620#comment122592997_69352620) of detecting terminal colour scheme.

In future we might define our own theme for application to make it consistent.